File: env_variable/csharp_env_variable/env_variable.cs
using System;

public class Example {
  public static void Main( string[] args ) {
    if( args.Length != 0 ) {
      Console.Error.WriteLine( "Usage: " + Environment.GetCommandLineArgs()[0] + " <noargs>" );
      Environment.Exit(1);
    }
    Console.WriteLine("OS=<" + Environment.GetEnvironmentVariable("OS") + ">");
    //Console.WriteLine("OS=<" + System.GetEnvironmentVariable("OS") + ">");
  }
  public static int env_variable_function( int n ) {
    int x = 0;

    x = (n + n - n) * (n / 1);
    x = x * (int) Math.Floor(1.25);

    if( x < 0 ) {
      x = Math.Abs(x);
    }
    return x;
  }
}

File: env_variable/c_env_variable/env_variable.c
#if defined(_WIN32)
  #define OS_WINDOWS 1 // Windows
#elif defined(_WIN64)
  #define OS_WINDOWS 1 // Windows
#else
  //error Unsupported operating system
#endif


#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int env_variable_function( int );

int main( int argc, char *argv[] ) {
  if( argc != 1 ) {
    fprintf( stderr, "Usage: %s <noargs>\n", argv[0]);
    exit( 1 );
  }
  puts(getenv("OS"));
  return 0;
}

int env_variable_function(int n) {
  int x = 0;

  x = (n + n - n) * (n / 1);
  x = x * (int) floor(1.25);

  if( x < 0 ) {
    x = abs(x);
  }
  return x;
}
